src/main/java/com/opentable/server/EmbeddedJettyBase.java

```diff
@@ -305,6 +312,17 @@ public void gracefulShutdown(ContextClosedEvent evt) {
         LOG.debug("Received application context closed event {}. Shutting down...", evt);
         LOG.info("Early shutdown of Jetty connectors on {}", container);
         if (container != null) {
+            if(shouldSleepBeforeShutdown) {
```

Comment:
Small stylistic nit  - I'd probably move most of this to a separate method
sleepForAWhile() or somesuch.